WrightTools/artists/_colors.py

```diff
@@ -133,6 +134,158 @@ def make_colormap(seq, name="CustomMap", plot=False):
+    use the r parameter instead for make_cubehelix.   Not used for ListedColormaps.
+
+    Parameters
+    ----------
+    cmap : matplotlib.colors.LinearSegmentedColormap
+        Colormap to be cycled.
+    rotations : int
+        number of rotations
+    eps:  difference between end of first cycle and beginning of next cycle
+        which may need to be specified if the colormap has many index values
+
+    Returns
+    -------
+    matplotlib.colors.LinearSegmentedColormap
+
+    """
+    colors = cmap._segmentdata
+    newmap = cmap.copy()
+
+    if isinstance(colors["red"], tuple):
+        re = colors["red"]
+        gr = colors["green"]
+        bl = colors["blue"]
+        try:
+            alphas = colors["alpha"]
+            lnal = len(alphas)
+        except:
+            lnal = 0
+            alphas = None
+
+        lnre = len(re)
+        lngr = len(gr)
+        lnbl = len(bl)
+
+        arre = list()
+        argr = list()
+        arbl = list()
+
+        for i in range(lnre):
+            arr = list(re[i])
+            arre.append(arr)
+        for i in range(lngr):
+            arr = list(gr[i])
+            argr.append(arr)
+        for i in range(lnbl):
+            arr = list(bl[i])
+            arbl.append(arr)
+
+        reds = np.array(arre, dtype=float)
+        greens = np.array(argr, dtype=float)
+        blues = np.array(arbl, dtype=float)
+        if lnal != 0:
+            arral = list()
+            for i in range(lnal):
+                arr = list(alphas[i])
+                arral.append(arr)
+            alphas = np.array(arral, dtype=float)
+    else:
+        reds = np.array(colors["red"], dtype=float)
+        greens = np.array(colors["green"], dtype=float)
+        blues = np.array(colors["blue"], dtype=float)
+        try:
+            alphas = np.array(colors["alpha"], dtype=float)
+            lnal = 1
+        except:
+            alphas = None
+            lnal = 0
+
+    redvec = reds[:, 0][:, None]
+    greenvec = greens[:, 0][:, None]
+    bluevec = blues[:, 0][:, None]
+
+    redmat = reds[:, 1:]
+    greenmat = greens[:, 1:]
+    bluemat = blues[:, 1:]
+
+    redf = reds[-1, :][None, :]
+    greenf = greens[-1, :][None, :]
+    bluef = blues[-1, :][None, :]
+
+    redvecr = redvec / rotations
+    greenvecr = greenvec / rotations
+    bluevecr = bluevec / rotations
+
+    redveca = redvecr
+    greenveca = greenvecr
+    blueveca = bluevecr
+
+    if lnal != 0:
+        alphavec = alphas[:, 0][:, None]
+        alphamat = alphas[:, 1:]
+        alphaf = alphas[-1, :][None, :]
+        alphavecr = alphavec / rotations
+        alphaveca = alphavecr
+
+    for i in range(rotations):
+        if i == 0:
+            red = np.append(redveca, redmat, axis=1)
+            green = np.append(greenveca, greenmat, axis=1)
+            blue = np.append(blueveca, bluemat, axis=1)
+            newmap._segmentdata["red"] = red
```

Comment:
you are modifying hidden attributes (`_name`) which is not a supported aspect of matplotlib colormaps--they are subject to change without notice.  You should rely on the public properties of the colormaps to generate this repetition.  

I think this is also the reason your method is specific to the segmented type of colormaps?